networking/using-ptp.adoc

```diff
@@ -45,19 +50,29 @@ include::modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc[lev
 
 include::modules/cnf-troubleshooting-common-ptp-operator-issues.adoc[leveloffset=+1]
 
-include::modules/cnf-about-ptp-and-clock-synchronization.adoc[leveloffset=+1]
+== PTP hardware fast event notifications framework
+
+:FeatureName: PTP events with boundary clock
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+[TIP]
```

Comment:
This is the same general comment as up above. Though I am not sure how to solve the problem of having the admonitions right next to each other in this case.